### PR TITLE
Add dockerfile for creating build environment

### PR DESCRIPTION
**Pull request by gfr10598**
Adds a docker file that produces a 32 bit docker image capable of building mlab NDT slice.

travis integration is also working in another repository, and will be introduced here shortly in a followup PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+Docker file for creating mlab builder images.
+
+### Building
+To build the docker image and tag it as m-lab/builder
+
+    docker build -t m-lab/builder .
+
+### Running the build environment
+
+    docker run m-lab/builder
```
Comment (pboothe) [inline README.md]: maybe `docker run -it m-lab/builder` ? That way ctrl-C will do the right thing.

File: dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+LABEL vendor="measurement-lab" description="Docker for building 32 bit mlab slices"
+
+RUN linux32 yum -y update
+RUN linux32 yum install -y wget git svn
+RUN linux32 yum install -y binutils qt gcc make patch libgomp
+RUN linux32 yum install -y glibc-headers glibc-devel kernel-headers kernel-devel htop dkms
+RUN linux32 yum install -y rpm-builder rpm-build m4 python-devel openssl-devel
+RUN linux32 rpm -ivh http://download.fedoraproject.org/pub/epel/6/i386/epel-release-6-8.noarch.rpm
+RUN linux32 yum install -y --nogpgcheck jansson-devel
+RUN linux32 yum install -y nodejs npm --enablerepo=epel
```
Comment (pboothe) [inline dockerfile]: Do all these `run linux32 yum install XYZ` lines need to happen separately or can they be combined?

File: README.md (hunk 1 above)
Comment (gfr10598) [inline README.md]: Done.  (D'oh)

File: dockerfile (hunk 2 above)
Comment (gfr10598) [inline dockerfile]: Many can be combined, but several have different flags.  I've combined two short ones, but left the others.  Prefer to leave them in small chunks, unless there is a reason to lump them together.

**Pull request merged**